src/test/datascience/interactive-common/randomBytes.vscode.test.ts

```diff
@@ -0,0 +1,50 @@
+import { ISystemPseudoRandomNumberGenerator } from '../../../client/datascience/types';
```

Comment:
Wait do we need to run this inside of VS code? Can't this just be a unit test?

Reply:
@DonJayamanne recommended running this in a VSCode test because we want to make sure this actually works, and across all platforms

Reply:
It's not actually using it inside of a window (which I'd argue one of the other tests will cover). It's just using the functional  test DataScienceIOC container.

A unit test would essentially be the same but would not need the container or the vscode bring up and runs on all platforms already.

Reply:
> A unit test would essentially be the same but would not need the container or the vscode bring up and runs on all platforms already.

Agreed, if we dont need the DI container, then we can just use a unit test.

Reply:
DI makes it way easier to resolve the dependencies of this class 🤣 But yup will rewrite as unit test